pkg/utils/constants.go

```diff
@@ -66,4 +66,9 @@ const (
 // configuration constants for the S3 repository
 const (
 	DefaultS3RepoPrefix = "backups/vsphere-volumes-repo"
+)
+
+// Minimum velero version number to meet velero plugin requirement
+const (
+	VELERO_MIN_VERSION = "v1.3.0"
```

Comment:
Go standard for constants is camel case, please change this to VeleroMinVersion.

Reply:
Changed to camel case.